recipes-bsp/u-boot/u-boot-imx-tools_2020.04.bb

```diff
@@ -6,5 +6,4 @@ PROVIDES_append_class-native = " u-boot-tools-native"
 PROVIDES_append_class-nativesdk = " nativesdk-u-boot-tools"
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-COMPATIBLE_MACHINE = "(mx6|mx7|mx8)"
 COMPATIBLE_MACHINE_class-target = "(mx6|mx7|mx8)"
```

Comment:
Well, not really. It shouldn't infect other layers.

What issue this triggers for you?

Reply:
Without dropping that line, how a end user could run "bitbake u-boot-imx-tools-native", or how could they set DEPENDS = "u-boot-tools-native" while u-boot-imx-tools-native is the preferred provider of u-boot-tools-native? Since as a native recipe, it could not comply only to imx6,imx7,imx8 machines, it needs to comply with x86-64 machine.